spec/controllers/verify/usps_controller_spec.rb

```diff
@@ -25,6 +26,37 @@
 
       expect(response).to be_ok
     end
+
+    it 'redirects if the user has sent too much mail' do
+      usps_mail_service = Idv::UspsMail.new(user)
+      allow(usps_mail_service).to receive(:mail_spammed?).and_return(true)
+      allow(subject.idv_session).to receive(:address_mechanism_chosen?).
+        and_return(true)
+      get :index
+
+      expect(response).to redirect_to verify_review_path
```

Comment:
The reason is that the controller is creating its own instance of `Idv::UspsMail` and so the stubbing here has literally no effect.

Two options:

1. make the `usps_mail_service` a memoized property of the controller, so we can stub on `controller.usps_mail_service`
2. Just write the things to the DB that the mail service looks for, so we don't have to stub